### PR TITLE
Export constants

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

**Review by dlongley — APPROVED** [review]
Approving. @davidlehn are you ok with re-exporting constants here?

File: lib/index.js [hunk 1]
```diff
@@ -13,3 +13,6 @@ for(const {contexts} of contextModules) {
     jsonLdDocumentLoader.addStatic(contextUrl, context);
   }
 }
+
+const {constants} = vcRevocationListContext;
+export {constants};
```
Comment (davidlehn) [inline lib/index.js]: I think this can be a one-liner?
```suggestion
export const {constants} = vcRevocationListContext;
```

**Comment by davidlehn** [comment]
This *could* just re-export constants.  But that seems rather specific.  What is the use case here?  I thought the main purpose of these bedrock*context packages was just to setup the a document loader?  But now there is a need to get the data out too?  Is this a suggested global change to all similar packages?  What happens when there are multiple modules setup in one package?  Would a better approach be for the code that needs this to just pull out the constants directly?

**Comment by JSAssassin** [comment]
@davidlehn This is currently being used in `@bedrock/vc-issuer` https://github.com/digitalbazaar/bedrock-vc-issuer/blob/fea601f182633ca8cee2383b13fb06f97c2343aa/lib/issuer.js#L15-L16. My understanding was that since `@bedrock/vc-issuer` is a bedrock library we should get the constants to be used there from a @bedrock/foo contexts package. I also noticed some other @bedrock/foo contexts  packages exporting constants - for eg. https://github.com/digitalbazaar/bedrock-did-context/blob/main/lib/index.js#L17-L18, so I thought maybe it was fine to do that here too.  Don't know what the best approach here should be though.

**Comment by dlongley** [comment]
@davidlehn -- This isn't as clean as I'd like either, however, I'd say the use case here is that the application is making the choice for which contexts/constants are installed (or at least what the constants point to) ... which is a different decision than is usually made in a local file wrt. what it is directly importing. This is because it needs to be shared and consistent across the entire application. I'm going to merge these since we don't have a better way forward right now.

**Pull request merged**